pybricksdev/compile.py

```diff
@@ -123,12 +124,26 @@ async def compile_multi_file(path: str, abi: int):
     parts: List[bytes] = []
 
     for name, module in finder.modules.items():
+        if not module.__file__:
+            continue  # system module
```

Comment:
I'm not sure what a "system" module is, but I think the scope here is broader in any case, e.g. namespace modules won't have `__file__` attribute. So perhaps better to leave out the comment or just say that `__file__` attribute is optional.